### DISCUSSION TITLE
cant apply for the github College student developer pack

### DISCUSSION THREAD

**Discussion by asheshjyotii** [community/community#43137] [GitHub Education]
I already applied for it but didn't receive any response,but when i tried to do again it either shows 508 or 404 error please help

**Comment by DSDmark** [comment]
Hi @asheshjyotii,

Here are a few things you can try:

 -   Make sure you are using a supported web browser and that it is up to date. The GitHub Student Developer Pack application may not work correctly on older or unsupported browsers.

-    Check your internet connection to ensure that it is stable and working properly.

 -   If you are seeing a 508 error, this could be caused by a server-side issue. In this case, you can try waiting a little while and then trying again later.

  -  If you are seeing a 404 error, this could be caused by a problem with the URL you are trying to access. Make sure you are using the correct URL and that there are no typos.

If none of these solutions help, you can try contacting [**GitHub Support**](https://support.github.com/).

**Reply by asheshjyotii** [reply]
Thanks,but changing the browser didn't help.....it might be problem from github's side. Anyway thanks for your effort
